src/components/check-status-responses/CheckStatusShippingCanadaPost.tsx

```diff
@@ -34,6 +34,7 @@ export const CheckStatusShippingCanadaPost: FC<
             </p>
             <p>
               <ExternalLink
+                data-gc-analytics-exempt=""
```

Comment:
I feel like this would be a little more sense:

```suggestion
                data-gc-analytics-exempt={true}
```

Reply:
It is rendered at AA team asked for.
![image](https://github.com/DTS-STN/passport-status/assets/114004123/a91ff968-d8e7-4d9a-8bc4-391c9ffd54f1)

Reply:
Setting it to `true` is valid and more expressive than `""`

https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML

Reply:
Ok, will fix but I don't care either will do the job.